Update TimeTest.Scala

naqvisat opened the pull request:
Unlocked two testcases operators and now

naqvisat committed:
Update TimeTest.Scala

Unlocked two testcases operators and now

tindzk commented on shared/src/test/scala/pl/metastack/metatime/TimeTest.scala:
```diff
@@ -18,13 +18,13 @@ object TimeTest extends SimpleTestSuite with TestIgnored {
-  testIgnored("now") {
-    assertEquals(Time.now(), ???)
+  test("now") {
+    assertEquals(Time(0,0,0,0) < Time.now(), true)
```
You could call `Time.now().unix()`, then wait a second and call it again. Afterwards check whether the difference is `<= 1.second.unix()`.

naqvisat committed:
Updated test assertions

Updated test assertions for Opertors and set "now" to ignored and added a new test "plusMinusTime" to do in future.

naqvisat commented:
Not needed any more because I just opened a new Pull request based on updated branch.
